src/core/src/local_loggers/CompositeLogger.py

```diff
@@ -48,32 +51,74 @@ def log(self, message, message_type=Constants.TelemetryEventLevel.Informational)
             timestamp = self.env_layer.datetime.timestamp()
             self.file_logger.write("\n" + timestamp + "> " + message.strip(), fail_silently=False)
 
-    def log_error(self, message):
-        """log errors"""
+    def flush(self, message, message_level):
+        if message_level == Constants.MessageLevel.ERROR:
```

Comment:
This is not needed if you re-use telemetry log level. You could simple call log method with appropriate enum here.